libvast/vast/detail/partition_common.hpp

```diff
@@ -50,8 +51,26 @@ fetch_indexer(const PartitionState& state, const meta_extractor& ex,
         row_ids |= ids;
     }
   } else if (ex.kind == meta_extractor::age) {
-    for (const auto& [_, ids] : state.type_ids())
-      row_ids |= ids;
+    // For the passive partition we can trust the eval from the meta index, but
+    // we cannot rely on that for the active partition. We create a time
```

Comment:
That was a bit confusing to understand, I'd suggest something like "For a passive partition, this already went through a time synopsis in the meta index, but for the active partition we create an ad-hoc time synopsis here to do the lookup."